### ISSUE TITLE
Management Client-Side Libraries

### ISSUE THREAD

**Issue by DmitriyKulagin** [nopSolutions/nopCommerce#4240]
In order to timely update the jquery libraries used in the project, we need a tool that would automate this process to some extent.

**Comment by iyilm4z** [comment]
@DmitriyKulagin how about [LibMan](https://docs.microsoft.com/tr-tr/aspnet/core/client-side/libman/libman-vs?view=aspnetcore-3.1)? That's Microsoft's recommended solution for client side libraries.

**Comment by DmitriyKulagin** [comment]
@iyilm4z  Yes, of course LibMan is a good tool, and now plans to use it. But it will not be used alone, as it is designed for simple libraries in which a small number of files. When it comes to more extensive libraries, for example kendo-ui-core, it is more correct to use the NPM package manager.

**Comment by sefacan** [comment]
You can use BuildBundlerMinifier library.

With bundleconfig.json like this
```
{
    "outputFileName": "wwwroot/bundles/site.min.css",
    "inputFiles": [
      "wwwroot/plugins/bootstrap-datepicker/css/datepicker.css",
      "wwwroot/plugins/bootstrap-timepicker/css/bootstrap-timepicker.css",
      "wwwroot/plugins/bootstrap-multiselect/css/bootstrap-multiselect.css",
      "wwwroot/plugins/boostrapv3/css/bootstrap.min.css",
      "wwwroot/plugins/boostrapv3/css/bootstrap-theme.min.css",
      "wwwroot/plugins/font-awesome/css/font-awesome.css",
      "wwwroot/css/style.css",
      "wwwroot/css/responsive.css",
      "wwwroot/css/custom-icon-set.css"
    ]
  },
  {
    "outputFileName": "wwwroot/bundles/common.min.js",
    "inputFiles": [
      "wwwroot/plugins/jquery-1.8.3.min.js",
      "wwwroot/js/jquery.validate.min.js",
      "wwwroot/js/jquery.validate.unobtrusive.min.js",
      "wwwroot/plugins/jquery-ui/jquery-ui-1.10.1.custom.min.js",
      "wwwroot/plugins/boostrapv3/js/bootstrap.min.js"
    ],
    "minify": {
      "enabled": true,
      "renameLocals": true
    },
    "sourceMap": false
  }
```
PageHeadBuilder uses high CPU and Memory. You must get rid of it.

**Comment by a-patel** [comment]
Nop should use npm package manager.

**Comment by DmitriyKulagin** [comment]
Closed #4240
